DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -0,0 +1,156 @@
+//
+//  attachmentItemScript.js
+//
+//  This script is a simplified version of the original attachmentItemScript.js 
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function() {
+    var ATTACH_SOUND = SoundCache.getSound(Script.resolvePath('sound/attach_sound_1.wav'));
+    var DETACH_SOUND = SoundCache.getSound(Script.resolvePath('sound/detach.wav'));
+
+    var LEFT_RIGHT_PLACEHOLDER = '[LR]';
+    var ATTACH_DISTANCE = 0.35;
+    var DETACH_DISTANCE = 0.5;
+    var AUDIO_VOLUME_LEVEL = 0.2;
+
+    var TRIGGER_INTENSITY = 1.0;
+    var TRIGGER_TIME = 0.2;
+
+    var EMPTY_PARENT_ID = "{00000000-0000-0000-0000-000000000000}";
+
+    var MESSAGE_CHANNEL_BASE = "AvatarStoreObject";
+    var messageChannel;
+    
+    var _this, _entityID;
+    var _attachmentData;
+    var _supportedJoints = [];
+    var isAttached;
+
+    var firstGrab = true;
+
+    /**
+     * 
+     * @param {Object} entityProperties 
+     * @param {touchJSONUserDataCallback} touchCallback 
+     */
+    var touchJSONUserData = function(entityProperties, touchCallback) {
+        try {
+            // attempt to touch the userData
+            var userData = JSON.parse(entityProperties.userData);
+            touchCallback.call(this, userData);
+            entityProperties.userData = JSON.stringify(userData);
+        } catch (e) {
+            print('Something went wrong while trying to touch/modify the userData. Could be invalid JSON or problem with the callback function.');
+        }
+    };
+
+    /**
+     * This callback is displayed as a global member.
+     * @callback touchJSONUserDataCallback
+     * @param {Object} userData
+     */
+
+    function AttachableItem() {
+        _this = this;
+    }
+    AttachableItem.prototype = {
+        preload : function(entityID) {
+            _entityID = entityID;
+            var properties = Entities.getEntityProperties(entityID);
```

Comment:
Please filter this to only request the parentID and userData    (parentID is probably requested by default, but lets still do it in case that changes in the future)